fpdf/template.py

```diff
@@ -147,6 +147,29 @@ def _parse_multiline(s):
             return False
         return None
 
+    def parse_json(self, infile, encoding="utf-8"):
+        """
+        Load the template definition from a JSON file.
+        The data must be structured as an array of objects, with names and values exactly
+        equivalent to what would get supplied to load_elements(),
+
+        Arguments:
+
+            infile (string): The filename of the CSV file.
+
+            encoding (string): The character encoding of the file. Default is UTF-8.
+        """
+        with open(infile, encoding=encoding) as f:
```

Comment:
It could be nice to have this method also accept [pathlib.Path](https://docs.python.org/3/library/pathlib.html#pathlib.Path) objects.

I think it should as simple as passing `str(infile)` to open() there,
but maybe we could have a simple unit test for this case?

Reply:
Since both methods use the builtin `open()`, they already accept all "path-like objects", which includes `pathlib.Path`.

If I understand the documentation correctly, we should then use an `os.PathLike` as a type hint.